feathr_project/test/test_azure_spark_e2e.py

```diff
@@ -83,23 +83,19 @@ def test_feathr_online_store_agg_features():
     # this part with the test_feathr_online_store test case
     client.wait_job_to_finish(timeout_sec=Constants.SPARK_JOB_TIMEOUT_SECONDS)
 
-    res = client.get_online_features(online_test_table, '265', [
+    res = client.get_online_features(online_test_table, ["81", "254"], [
```

Comment:
Is there existing test cases that can test get online features by single key? If not, suggest add a new test case instead of updating existing get by single key test case

Reply:
Yes, we have some test cases in 'test_feature_materialization.py' and 'test_local_spark_e2e.py' that can test get online features by single key.